internal/sidekick/internal/parser/discovery/services_test.go

```diff
@@ -44,13 +44,50 @@ func TestService(t *testing.T) {
 				Documentation: "Returns the specified Zone resource.",
 				InputTypeID:   ".google.protobuf.Empty",
 				OutputTypeID:  "..Zone",
+				PathInfo: &api.PathInfo{
+					Bindings: []*api.PathBinding{
+						{
+							Verb: "GET",
+							PathTemplate: api.NewPathTemplate().
+								WithLiteral("compute").
+								WithLiteral("v1").
+								WithLiteral("projects").
+								WithVariableNamed("project").
+								WithLiteral("zones").
+								WithVariableNamed("zone"),
+							QueryParameters: map[string]bool{},
+						},
+					},
+					BodyFieldPath: "*",
+				},
 			},
 			{
 				ID:            "..zones.list",
 				Name:          "list",
 				Documentation: "Retrieves the list of Zone resources available to the specified project.",
 				InputTypeID:   ".google.protobuf.Empty",
 				OutputTypeID:  "..ZoneList",
+				PathInfo: &api.PathInfo{
+					Bindings: []*api.PathBinding{
+						{
+							Verb: "GET",
+							PathTemplate: api.NewPathTemplate().
+								WithLiteral("compute").
+								WithLiteral("v1").
+								WithLiteral("projects").
+								WithVariableNamed("project").
+								WithLiteral("zones"),
+							QueryParameters: map[string]bool{
+								"filter":               true,
+								"maxResults":           true,
+								"orderBy":              true,
+								"pageToken":            true,
+								"returnPartialSuccess": true,
+							},
+						},
+					},
+					BodyFieldPath: "*",
```

Comment:
This seems counter to listing the query params. Isn't the body typically empty for GETs?

https://google.aip.dev/127

Reply:
I think we already have `*` as the body for GET requests. For example:

https://github.com/googleapis/librarian/blob/2835edabdba6002f8b4f091e456c113b7c3b452a/internal/sidekick/internal/sample/api.go#L167

https://github.com/googleapis/librarian/blob/2835edabdba6002f8b4f091e456c113b7c3b452a/internal/sidekick/internal/parser/protobuf_mixin_test.go#L230

I do not think we need to handle them specially here.